enterprise/server/remote_execution/execution_server/execution_server_test.go

```diff
@@ -0,0 +1,147 @@
+package execution_server
```

Comment:
nit: we usually prefer writing tests as part of a separate package, like `execution_server_test`, since it usually leads to higher quality tests as you're testing only using the public API of the package, which is how it will actually be used by other parts of the system in practice.

Reply:
Done